### PR TITLE
Separators components & examples

### LINKED ISSUE #1057: JSX: separators

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-13 at 11 05 22

Closes #1057

### CODE REVIEW

File: src/components/separators/HorizontalSeparator.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const HorizontalSeparator = ({spaced, short}) => {
```
Comment (matikrk) [inline src/components/separators/HorizontalSeparator.jsx]: Maybe HorizontalSeparator -> SeparatorHorizontal

File: src/components/separators/Separator.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const SIZE = {
+  NORMAL: 'normal',
+  SMALL: 'small',
+  LARGE: 'large'
+};
+
+const Separator = ({size = SIZE.NORMAL}) => {
```
Comment (matikrk) [inline src/components/separators/Separator.jsx]: IMHO better change to SeparatorVertical
and then change styles from `sg-separator` to `sg-vertical-separator` - there is only 6 occurrence in  our projects so we can change it easily and have better described naming
WDYT @aju @kdzwinel @burakukula @matzimowski @vergilius

File: src/components/separators/HorizontalSeparator.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  const separatorClass = classNames('sg-horizontal-separator', {
+    'sg-horizontal-separator--spaced': spaced && !short,
+    'sg-horizontal-separator--short-spaced': spaced && short
```
Comment (matikrk) [inline src/components/separators/HorizontalSeparator.jsx]: only short do nothing and can be confused, maybe TYPE:{normal, spaced, short-spaced}

File: src/components/separators/HorizontalSeparator.jsx (hunk 1 above)
Comment (anna-wro) [inline src/components/separators/HorizontalSeparator.jsx]: good point 👍

**Comment by anna-wro** [comment]
* Separator -> SeparatorVertical
* HorizontalSeparator -> SeparatorHorizontal

File: src/components/separators/Separator.jsx (hunk 2 above)
Comment (matzimowski) [inline src/components/separators/Separator.jsx]: :+1: for this change ❤️

**Comment by anna-wro** [comment]
I renamed .sg-separator to .sg-vertical-separator
(I can change it in all our projects, there are only 6 occurrences)

File: src/components/content-box/pages/content-box.jsx [hunk 4]
```diff
@@ -103,7 +104,7 @@ const examplePart2 = <ContentBox>
         <Sticker type={STICKER_TYPE.ANSWER}/>
       </div>
     </div>
-    <div className="sg-separator"></div>
+    <div className="sg-vertical-separator"></div>
```
Comment (aju) [inline src/components/content-box/pages/content-box.jsx]: why not jsx? :)

Comment (anna-wro) [inline src/components/content-box/pages/content-box.jsx]: I've just changed it, thanks for noticing :)

**Pull request merged**